packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -613,6 +615,11 @@ export default {
                     timeout: this.checkoutTimeout
                 });
 
+                if (this.message && this.message.code === ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE) {
+                    await this.loadAvailableFulfilment();
+                    this.selectorKey++;
```

Comment:
Probably worth documenting somewhere why we're using this approach instead of just refreshing the list of elements, which, theoretically, should suffice. Maybe add a comment here?